src/command_modules/azure-cli-container/README.rst

```diff
@@ -0,0 +1,186 @@
+Microsoft Azure CLI 'container' Command Module
+==============================================
+
+Commands to manage Azure container instances
++++++++++++++++++++++++++++++++++++++++++++++
+::
+
+    Group
+        az container: Manage Azure Container Instances.
+
+    Commands:
+        create : Create or update container group.
+        delete : Delete a container group.
+        list   : List container groups by resource group.
+        listall: List all container groups.
```

Comment:
`list` and `listall` should be folded into a single list command that makes the appropriate API call based on whether resource group is supplied or not.